spec/spec_helper_acceptance.rb

```diff
@@ -1,22 +1,59 @@
-require 'bolt_spec/run'
-require 'beaker-task_helper/inventory'
-require 'beaker-pe'
-require 'beaker-puppet'
-require 'puppet'
-require 'beaker-rspec'
-require 'beaker/puppet_install_helper'
-require 'beaker/module_install_helper'
+# frozen_string_literal: true
```

Comment:
use the latest version here, it adds native docker support https://github.com/puppetlabs/puppet_litmus/wiki/Converting-a-module-to-use-Litmus#specspec_helper_acceptancerb

Reply:
Thank you.Its updated.